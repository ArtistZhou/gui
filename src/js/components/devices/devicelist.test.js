import React from 'react';
import renderer from 'react-test-renderer';
import DeviceList from './devicelist';

it('renders correctly', () => {
  const tree = renderer.create(<DeviceList devices={[]} selectedRows={[]} columnHeaders={[1, 2, 3, 4]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
