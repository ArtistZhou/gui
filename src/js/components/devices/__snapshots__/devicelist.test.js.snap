// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="deviceList "
>
  <div
    className="flexbox header"
    style={
      Object {
        "padding": "0 12px",
      }
    }
  >
    <div
      className="columnHeader"
      style={
        Object {
          "width": "25%",
        }
      }
    />
    <div
      className="columnHeader"
      style={
        Object {
          "width": "25%",
        }
      }
    />
    <div
      className="columnHeader"
      style={
        Object {
          "width": "25%",
        }
      }
    />
    <div
      className="columnHeader"
      style={
        Object {
          "width": "25%",
        }
      }
    />
    <div
      style={
        Object {
          "width": 48,
        }
      }
    />
  </div>
  <div
    className="body"
  />
  <ul
    className="rc-pagination rc-pagination-simple margin-top"
    style={Object {}}
  >
    <li
      aria-disabled={true}
      className="rc-pagination-disabled rc-pagination-prev"
      onClick={[Function]}
      onKeyPress={[Function]}
      tabIndex={null}
      title="Previous Page"
    >
      <a
        className="rc-pagination-item-link"
      />
    </li>
    <li
      className="rc-pagination-simple-pager"
      title="undefined/NaN"
    >
      <input
        onChange={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        size="3"
        type="text"
      />
      <span
        className="rc-pagination-slash"
      >
        Ôºè
      </span>
      NaN
    </li>
    <li
      aria-disabled={true}
      className="rc-pagination-disabled rc-pagination-next"
      onClick={[Function]}
      onKeyPress={[Function]}
      tabIndex={null}
      title="Next Page"
    >
      <a
        className="rc-pagination-item-link"
      />
    </li>
  </ul>
</div>
`;
